riakcs/manifest.json

```diff
@@ -3,7 +3,7 @@
     "data store"
   ],
   "creates_events": false,
-  "display_name": "Riak CS",
+  "display_name": "RiakCS",
```

Comment:
```suggestion
  "display_name": "Riak CS",
```

Reply:
Made the revisions to dashboards. Unfortunately, this change is not trivial as it has to match what was previously in dogweb (which is `RiakCS`), so we cannot change this. However, these values that cannot change are not public facing and we can edit the public facing title of the integration to make `Riak CS`. I will do this in another PR :)